reading/structs.livemd

```diff
@@ -76,7 +76,8 @@ all of them.
 One main reason, is to validate our data and ensure keys and values exist with the expected data.
 
 For example, using a struct guarantees that certain keys exist, even if the value is `nil`. So it's
-always safe to use `.` syntax. Notice how `map.name` results in an error, but `struct.name` simply returns nil.
+always safe to use `.` syntax. Notice how `map.name` results in an error, but `struct.name` simply 
+returns `nil`.
```

Comment:
Livebook now wraps markdown content (Wahoo!) so it's no longer necessary to break up markdown content for the sake of readability.

That said, I'm not going to block PRs for line breaks, just wanted to let you know!

Thank you as always for your hard work @kianmeng you've been a tremendous help on this project. Please feel free to message me anytime if I can help or if you have ideas for the academy :)

Reply:
I see. I was following the line length of the previous sentence.

Reply:
There are lots of places that have these line breaks because I put them in before livebook made a change to how their markdown cells work, figured this was a good time to bring it up in case you notice them!